src/page-multi-facility/ScenarioLibraryModal.tsx

```diff
@@ -165,11 +182,50 @@ const ScenarioLibraryModal: React.FC<Props> = ({ trigger }) => {
   const [showDeleteModal, setShowDeleteModal] = useState(false);
   const rejectionToast = useRejectionToast();
 
+  const sortScenarios = (list: any) => {
```

Comment:
It should be possible to achieve this same sorting result using lodash's built in `orderBy` function in the same way we're using it here: https://github.com/Recidiviz/covid19-dashboard/blob/master/src/database/index.tsx#L224

You may have to tweak it a bit to get case insensitive sorting to work by following this example:  https://stackoverflow.com/questions/37848030/lodash-how-to-do-a-case-insensitive-sorting-on-a-collection-using-orderby

Reply:
I was trying to tweak the `orderBy` in my first attempt but was not aware of the lodash. Anyways, tried above suggestion and it seems to do the trick. Thanks!